loadtests/research/locustfile.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+from locust import  *
```

Comment:
Please do not use this was to import. Do import only what you need for the test.